### ISSUE TITLE
Will we get any update ?

### ISSUE THREAD

**Issue by Ud4ba** [greenshot/greenshot#263]
The apps seems outdated since 2017 , will we get any updates ?

**Comment by Lakritzator** [comment]
Yes, you can follow the progress here: https://github.com/greenshot/greenshot/projects/2

Any specific reason?

I had some issues with my development gear, unfortunately professional development gear doesn't come cheap and with the pandemic there was a supply shortage for > 6 months. I also don't have as much free time, with the work that puts food on the table and a kid who I need to occupy most of my "free" time, so it will takes time.

**Comment by EPuncker** [comment]
can we get a new unstable release? :D
thank you so much for your effort on this amazing tool

**Comment by Lakritzator** [comment]
@EPuncker Although we use the name "unstable" to make sure we cannot guarantee a fully functional version, right now I'm in the middle of transforming the cloud services so they are not broken due to an embedded browser and that would break it for everybody who uses this. As soon as I have this under control, I'm more than happy to put automated publishing on again.

I probably can speed up the process of publishing parallel, but this would slow me down else-where...

@ All: There is a technical list here, but in our Jira issue tracker I am still working on making a clear 1.3 release so you can see what is included.

**Comment by Lakritzator** [comment]
@EPuncker just released a new build, to be honest I didn't put much time in testing it as it took quite long to make it possible.
There will probably be issues with cloud services....

**Comment by EPuncker** [comment]
> @EPuncker just released a new build, to be honest I didn't put much time in testing it as it took quite long to make it possible.
> There will probably be issues with cloud services....

thank you so much, I mostly only use local save and imgur, I think I'll be fine :D

**Comment by Lakritzator** [comment]
@EPuncker Not sure about Imgur, I'm improving all the cloud services and that belongs in this category.
If it doesn't work, go back to 1.2.10

**Comment by EPuncker** [comment]
> @EPuncker Not sure about Imgur, I'm improving all the cloud services and that belongs in this category.
> If it doesn't work, go back to 1.2.10

it is working, just so you know, thank you
